### PR TITLE
Update da.xml

### PR DESCRIPTION
**Pull request by thomashdk**
Found som missing translations

### CODE REVIEW

**Comment by abjerner** [comment]
Thanks @thomashdk and congrats with your first contribution 👍 

Do you by any chance have an example of a context in which these labels are used? That would make teasting a bit easier for me.

**Comment by thomashdk** [comment]
Hey ! Thanks 

Finally took time to find out how to do it... :D

My user is set to **Danish**

And when I will "Save and Publish" Or Just "Save" and node the text will appear.

image

I will of course in the future be better to remember to write how to reproduce an error :)

File: src/Umbraco.Web.UI/Umbraco/config/lang/da.xml [hunk 1]
```diff
@@ -264,6 +264,8 @@
     <key alias="titleOptional">Titel (valgfri)</key>
     <key alias="altTextOptional">Alternativ tekst (valgfri)</key>
     <key alias="type">Type</key>
+    <key alias="variantsToPublish">Hvilke varianter vil du publicere?</key>
```
Comment (abjerner) [inline src/Umbraco.Web.UI/Umbraco/config/lang/da.xml:267]: The other Danish labels in the say context says _udgive_, so could you use that as well here?

```suggestion
    <key alias="variantsToPublish">Hvilke varianter vil du udgive?</key>

```

File: src/Umbraco.Web.UI/Umbraco/config/lang/da.xml [hunk 2]
```diff
@@ -264,6 +264,8 @@
     <key alias="titleOptional">Titel (valgfri)</key>
     <key alias="altTextOptional">Alternativ tekst (valgfri)</key>
     <key alias="type">Type</key>
+    <key alias="variantsToPublish">Hvilke varianter vil du publicere?</key>
+    <key alias="variantsToSave">Vælg hvilke varianter der skal gemmes.</key>
```
Comment (abjerner) [inline src/Umbraco.Web.UI/Umbraco/config/lang/da.xml:268]: Could you add a comma here?
```suggestion
    <key alias="variantsToSave">Vælg hvilke varianter, der skal gemmes.</key>

```

**Review by abjerner — CHANGES_REQUESTED** [review]
Great. I've highlighted some minor adjustments. If you can make those, I'd be happy to merge 😉

File: src/Umbraco.Web.UI/Umbraco/config/lang/da.xml (hunk 2 above)
Comment (abjerner) [reply src/Umbraco.Web.UI/Umbraco/config/lang/da.xml:269]: @thomashdk can you make this change as well? Then it should be ready to merge.

**Review by abjerner — APPROVED** [review]

**Comment by abjerner** [comment]
Hey @thomashdk. I didn't hear back from you, so I went ahead and added the change that I had suggested, and merged the entire PR into `v8/contrib`. Then the translations should hopefully make it's way to the next Umbraco release 🎉 👍 

Do you by any chance have a profile on [Our Umbraco](https://our.umbraco.com/)? As a token of appreciation, and if you share the link to your Our profile here, we'll add a shiny contributors badge to your profile 😉

**Comment by thomashdk** [comment]
Hej @abjerner  Thanks ! 

My profile is https://our.umbraco.com/members/id:271846/ :)

**Comment by nul800sebastiaan** [comment]
Sorry for the delay @thomashdk and congrats with your shiny new contributor badge on Our! https://our.umbraco.com/members/id:271846/ - Thanks again! 🥇

**Pull request merged**